scripts/check-vsix-size.ts

```diff
@@ -12,7 +12,7 @@ const vsixFileName = path.resolve(
 );
 const size = fs.statSync(vsixFileName).size;
 
-const maxSize = 8 * 1000000; // 8 MB
+const maxSize = 8.5 * 1000000; // 8.5 MB
```

Comment:
```suggestion
const maxSize = 8_500_000; // 8.5 MB
```

(might not even need the comment anymore then :))